content/Watsonx/Assistant/104.md

```diff
@@ -1,6 +1,7 @@
 ---
+title: '104: Analytics'
 timeToComplete: 30
-updated: 2023-03-31
+updated: 2024-02-06
 ---
 
```

Comment:
Include quiz alert. 

<QuizAlert text='Heads Up! Quiz material will be flagged like this!' />

Reply:
added

Reply:
Heads Up! Quiz material will be flagged like this!

Please add above line on top of the page.